lib/command/CommandClient.js

```diff
@@ -187,6 +187,8 @@ class CommandClient extends Client {
                     }
                 }
             }
+        } else {
```

Comment:
Yeah, this should be on the previous line if you only wait it for valid prefixes. Currently this runs if the prefix does not match too.